bin/fullstack.rb

```diff
@@ -22,6 +21,11 @@
 # Run CSS generator
 rails_command 'css:install:tailwind'
 
+# Add Sass build for Active Admin styles
+insert_into_file 'package.json', after: "application.css --minify" do
+  ' && sass ./app/assets/stylesheets/active_admin.scss:./app/assets/builds/active_admin.css --no-source-map --load-path=node_modules'
```

Comment:
it'd be better to have
`sass ./app/assets/stylesheets:./app/assets/builds --no-source-map --load-path=node_modules`